assets/react/v3/entries/course-builder/components/curriculum/TopicContent.tsx

```diff
@@ -123,58 +140,86 @@ const TopicContent = ({ type, topic, content, isDragging = false, onCopy, onDele
   };
 
   return (
-    <div css={styles.wrapper({ isDragging })} ref={setNodeRef} style={style} {...attributes}>
-      <div css={styles.iconAndTitle({ isDragging })} {...listeners}>
-        <div data-content-icon>
-          <SVGIcon
-            name={icon.name as IconCollection}
-            width={24}
-            height={24}
-            style={css`
+    <>
+      <div
+        css={styles.wrapper({ isDragging, isMeetingSelected: meetingType === type })}
+        ref={setNodeRef}
+        style={style}
+        {...attributes}
```

Comment:
Use spread `{...attributes}` at the beginning. Otherwise it could overwrite the ref, style etc.